docs/pages/guides/spend-permissions/quick-start.mdx

```diff
@@ -2,22 +2,26 @@ import { Callout } from "vocs/components";
 
 # Getting Started with Spend Permissions
 
-This guide will walk you through a simple example of building an app that leverages Spend Permissions using 
+This guide will walk you through a simple example of building an app that leverages Spend Permissions using
 [OnchainKit](https://onchainkit.xyz/), [Viem](https://viem.sh/) and [Wagmi](https://wagmi.sh/).
 
-A complete example of this demo can be found [here](https://github.com/ilikesymmetry/subscribe-onchain).  
+A complete example of this demo can be found [here](https://github.com/ilikesymmetry/spend-permissions-quickstart).  
```

Comment:
new repo name to be more clear that it is meant to be a complete version of this quickstart guide